code/src/index.css

```diff
@@ -1,13 +1,177 @@
+:root {
+  --main-color-:#ffff;
+  --secondary-color: #a0a0a0;
+}
+
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+ background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+hr {
+  border: 1px solid #212121;
+}
+
+h2 {
+  margin: 0px;
+ 
+}
+
+h1{
+  margin: 0px;
+}
+
+.main-container {
+  padding: 10px;
+  font-family: Helvetica;
+  align-items: center;
+  color: white;
+}
+
+.title {
+  font-weight: bold;
+  color:#ffff;
+  margin: 20px 20px 20px 20px;
+  font-size: 14px;
+  font-family: Helvetica;
+}
+
+.album-image {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+  transition: filter 0.3s ease-in-out;
+}
+.album-image:hover {
```

Comment:
I believe  if you put a container that fits both the cover and the buttons, then position the buttons in to stay on top of the image and then add :hover upon that container and display that element on hover you might get sought after result. That is the way I would do it atleast with album cover/icon:hover